### PR TITLE
Enable CORS for GET API

### PR DESCRIPTION
**Pull request by louy2**
Tested locally that it is working.

### CODE REVIEW

**Comment by louy2** [comment]
Fix #80

File: yacs/settings/base.py [hunk 1]
```diff
@@ -394,3 +396,12 @@ def relative_path(*path):
...
 }
+
+# === corsheaders ===
+
+# Enable public access
+CORS_ORIGIN_ALLOW_ALL = True
+# only for api
+CORS_URLS_REGEX = r'^/api/.*$'
+# only for GET
+CORS_ALLOW_METHODS = ('GET')
```
Comment (jeffh) [inline yacs/settings/base.py]: I think either remove the parens here or add a comma, because this isn't a tuple in python as it currently is.

**Comment by jeffh** [comment]
Just a minor fix and then I can merge it in.

**Comment by louy2** [comment]
Comma added.

**Comment by jeffh** [comment]
Thanks @louy2!

**Comment by louy2** [comment]
You're welcome. When will this be live?

**Comment by jeffh** [comment]
It's live now, I just deployed it.

**Pull request merged**